Dockerfile

```diff
@@ -15,7 +15,7 @@
 # limitations under the License.
 #
 
-FROM golang:1.15
```

Comment:
You will also need to update Travis:
https://github.com/apache/openwhisk-wskdeploy/blob/master/.travis.yml